### ISSUE TITLE
Add Catalog Search Results 

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/allsearch_frontend#20]
## Description
Displays catalog results from our Orangelight application. 

### Labeling
What heading should be used for the component. 

### Acceptance Criteria

- [x] It looks like the mockup
- [x] Add the component(s) to to the general page layout template
- [x] Is part of the main page layout, located in the "first" column, first position. 
- [x] Displays a heading that is not a link
- [x] Display an icon next to the heading relevant to the component type (i.e a book for the catalog)
- [x] Provides a link to see more results in the source user interface positioned to the right of the title with the label "Refine"
- [x] Displays a description for the component (AUX to provide, can use lorum ipsum to start)
- [x] Displays up to three search results. 
- [ ] If there are no results display a message to the user that provides them with a link to the source system to go there easily if they want to go right to that tool. Text "No results found. Search the [insert link]Catalog[close link].
- [x] Includes the document ID of each result in a data attribute (probably attached to the html around the title field?)
- [x] Add a link at the bottom of the component to the source interface search results that displays the number of total results in the source interface.
- [x] Each Search Result Should display the following
    * Title Field that is also a link to the record in source service
    * Author or equivalent
    * Format type or equivalent
    * Icon relevant to format type
    * If the item has online materials an online icon and a link to the online resource (sorta blocked by https://github.com/pulibrary/allsearch_api/issues/125, but I could just mock it locally)
    * If a single print holding display the holding location name with call number, otherwise say "multiple copies" 
    * Publisher statement with publication year if available
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Implementation Notes
Adding availability badges is a possibility but would need to happen in a separate ticket. We should include an attribute where the record ID can easily be passed to another asynchronous action to make that call if we decide to implement this.
